tests/primer/primer_tool.py

```diff
@@ -202,16 +205,20 @@ def _create_comment(
                     print(message)
                     fatal_count += 1
                 else:
-                    filepath = str(message["path"]).replace(
-                        str(package_data.clone_directory), ""
-                    )
                     new_non_fatal_messages += (
                         f"{count}) {message['symbol']}:\n*{message['message']}*\n"
                         f"{package_data.url}/blob/{package_data.branch}{filepath}#L{message['line']}\n"
                     )
                     print(message)
                     count += 1
 
+            if new_astroid_errors:
+                comment += (
+                    "New error(s) were found stemming from the `astroid` library. "
+                    "This is unlikely to have been caused by your changes. "
+                    "(Pushing to this pull request after another commit has been pushed to main may resolve the issue.) "
```

Comment:
```suggestion
                    "(Pushing to this pull request after another commit has been pushed to main may resolve the issue.) "
```
I'm not sure I understand this. Do you mean rebasing on main after a fix is found and applied by a maintainer ?

Reply:
Ho right, did not think of that.